csharp/ql/lib/semmle/code/csharp/dispatch/RuntimeCallable.qll

```diff
@@ -24,13 +22,10 @@ class RuntimeCallable extends DotNet::Callable {
 
 /** A run-time method. */
 class RuntimeMethod extends RuntimeCallable {
-  RuntimeMethod() {
-    this instanceof Method or
-    this instanceof CIL::Method
-  }
+  RuntimeMethod() { this instanceof Method }
```

Comment:
This could be simplified to `class RuntimeMethod extends RuntimeCallable, Method`